README.md

```diff
@@ -1,8 +1,151 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams — including leading Layer 2s and rollups — to launch in minutes instead of months.
```

Comment:
```suggestion
Sindri provides automated ZK proving infrastructure, empowering hundreds of teams—including leading Layer 2s and rollups—to launch in minutes instead of months.
```
Nice use of em-dash, but in US English there's no space before or after the dash.